src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,39 +2,60 @@
 
 public class DefaultCountingOutRhymer {
 
-	private static final int _11 = 11;
+	private static final int FULL_STACK = 11;

```

Comment:
This constant is related to STACK_CAPACITY . Changing only one leads to an error (inconsistency).